builtin/providers/aws/resource_aws_db_instance_test.go

```diff
@@ -157,7 +146,6 @@ resource "aws_db_instance" "bar" {
 	password = "barbarbarbar"
 	username = "foo"
 
-	security_group_names = ["${aws_db_security_group.bar.name}"]
 	parameter_group_name = "default.mysql5.6"
 }
```

Comment:
Does this test pass? Does it require a security group at all? Or it can be launched without a security group?